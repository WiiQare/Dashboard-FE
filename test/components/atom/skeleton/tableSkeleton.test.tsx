import React from "react";
import { render } from "@testing-library/react";
import TableSkeleton from "../../../../src/components/atom/skeleton/tableSkeleton"; 

describe("TableSkeleton Component", () => {
    it("renders the correct number of skeleton cards", () => {
        const numberOfCards = 3; 
        const { container } = render(<TableSkeleton row={numberOfCards} />);

        const skeletonCards = container.querySelectorAll(".grid-cols-1");

        expect(skeletonCards).toMatchSnapshot();
    });
});
