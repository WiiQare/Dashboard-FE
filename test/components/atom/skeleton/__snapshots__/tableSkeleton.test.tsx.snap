// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableSkeleton Component renders the correct number of skeleton cards 1`] = `NodeList []`;
